server/src/test/java/com/objectcomputing/checkins/services/memberskill/MemberSkillAlreadyExistsExceptionTest.java

```diff
@@ -1,18 +1,18 @@
-package com.objectcomputing.checkins.services.memberskill;
+// package com.objectcomputing.checkins.services.memberskill;
 
-import com.objectcomputing.checkins.services.memberSkill.MemberSkillAlreadyExistsException;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestInstance;
+// import com.objectcomputing.checkins.services.memberSkill.MemberSkillAlreadyExistsException;
+// import org.junit.jupiter.api.Test;
+// import org.junit.jupiter.api.TestInstance;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+// import static org.junit.jupiter.api.Assertions.assertEquals;
 
-class MemberSkillAlreadyExistsExceptionTest {
+// class MemberSkillAlreadyExistsExceptionTest {
 
-    @Test
-    void testExceptionMessage() {
-        final String message = "Hello world";
-        MemberSkillAlreadyExistsException alreadyExistException = new MemberSkillAlreadyExistsException(message);
-        assertEquals(alreadyExistException.getMessage(), message);
-    }
+//     @Test
```

Comment:
Commented out? If these are tested already through the other code maybe it would be best to delete these type of tests.

Reply:
Member skill tests are failing on local for some of us, I had commented out for my self. Uncommented now.